src/lib/THERMAL/devThermal.cpp

```diff
@@ -55,7 +85,7 @@ static void timeoutThermal()
     if(OPT_HAS_THERMAL_LM75A && !CRSF::IsArmed() && connectionState != wifiUpdate)
```

Comment:
OFF TOPIC - Can anyone tell me what this is supposed to do with IsArmed? I'm not sure how the LM75 thresholds interact with the fan and is it possible we can remove this IsArmed check easily?

Reply:
I'm not sure why it wouldn't to thermal checking when armed or in wifi mode!? Colour me confuzzled 🤷‍♂️ 
This came from the original Axis PR, so I'll check on the Axis module what it's doing.

Reply:
I've tracked it back to the original PR before we had the device framework, where it would have been running on the main thread along with everything else and was causing problems with the SPI for the radio. Back then it was also using software SPI for the TFT so things were very slow!

TL;DR I believe these checks can be removed.